hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/LoadBalancer.java

```diff
@@ -147,4 +147,17 @@
 
   /*Updates balancer status tag reported to JMX*/
   void updateBalancerStatus(boolean status);
+
+  /**
+   * In some scenarios, Balancer needs to update internal status or information according to the
+   * current tables load
+   * TODO: Maybe we can make a better name than 'updateBalancerLoadInfo'.
```

Comment:
I would remove this TODO. It's an IA.Private interface, so we could change this if we want or just leave it. I can't think of a better name right now

Reply:
In terms of the name, as implemented this method is only called in the case where the balancer does not run. So calling it updateBalancerLoadInfo makes it seem like it might be called more generically. I wonder if you could call this method at the start of balanceTable and this would unify with the normal balance call path as well as requested in my other comment.